x-pack/docs/en/security/securing-communications/setting-up-ssl.asciidoc

```diff
@@ -19,21 +18,26 @@ applicable to your cluster.
 certificate and enable TLS on the transport layer. You can also optionally
 enable TLS on the HTTP layer. See
 {ref}/configuring-tls.html#tls-transport[Encrypting Communications Between Nodes in a Cluster] and
-{ref}/configuring-tls.html#tls-http[Encrypting HTTP Client Communications]. 
+{ref}/configuring-tls.html#tls-http[Encrypting HTTP Client Communications].
 
-. Configure the {monitor-features} to use encrypted connections. See <<secure-monitoring>>.
++
```

Comment:
should we also hint to the fact that TLS on transport layer is mandatory, contrary to TLS on http being optional ?